### PR TITLE
Feature/basic cookies session

### PR DESCRIPTION
**Pull request by jayesh315thinkitive**

### CODE REVIEW

File: basic-cookies/app.js
Comment (imtiyazthink) [inline basic-cookies/app.js:1]: this folder is repeared, please remove it

**Review by imtiyazthink — CHANGES_REQUESTED** [review]

File: basic-cookies-session/session/app.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    resave : false
+}))
+
+const user = {
+    name: "Jayesh",
+    Roll_number: 43,
+    Address: "Pune",
+}; 
+
+app.get("/login", (req, res) => {
```
Comment (imtiyazthink) [inline basic-cookies-session/session/app.js:23]: login route should have post method

**Review by imtiyazthink — CHANGES_REQUESTED** [review]

File: basic-cookies/app.js
Comment (jayesh315thinkitive) [reply basic-cookies/app.js:1]: Updated

File: basic-cookies-session/session/app.js (hunk 1 above)
Comment (jayesh315thinkitive) [reply basic-cookies-session/session/app.js]: Updated

**Review by imtiyazthink — APPROVED** [review]

**Pull request merged**